### ISSUE TITLE
BGM problem

### ISSUE THREAD

**Issue by RuiqingTang** [RVC-Project/Retrieval-based-Voice-Conversion#35]
When I tried to run `rvc infer -m {model.pth} -i {input.wav} -o {output.wav}` to convert my audio, I found that the background music in the audio would also be converted.

**Comment by alcoftTAO** [comment]
Yeah, everything in the audio is also converted, including BGM. Try to use an AI to separate the music from the vocals.
